spec/features/user/user_auth_spec.rb

```diff
@@ -62,10 +64,12 @@
   context 'when an admin signs in' do
     before :each do
       admin = FactoryGirl.create(:admin)
-      visit login_path
-      fill_in("session_email", with: admin.email)
-      fill_in("session_password", with: admin.password)
-      click_button 'Sign in'
+      visit root_path
+      within("#signInModal") do
+        fill_in("session_email", with: admin.email)
+        fill_in("session_password", with: admin.password)
+        click_button 'Sign in'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.